*** tests/A-01-33-M-ok.out	2024-10-18 19:27:35.678249000 +0100
--- tests/A-01-33-M-ok.outhyp	2024-10-22 17:39:17.933227017 +0100
***************
*** 18,30 ****
  5 - Plantar árvore
  6 - Mostrar árvores de habitat
  0 - Sair
! Escolha uma opção: HABITAT|H1|Aldeia dos Macacos|20|3
! ÁRVORE|T1|Pinheiro 1|5|20|PERENE|GERARFOLHAS
! ÁRVORE|T2|Oliveira|1200|10|PERENE|GERARFOLHAS
! ÁRVORE|T4|Pinheiro 4|60|20|PERENE|GERARFOLHAS
! HABITAT|H2|Aldeia dos Ursos|20|1
! ÁRVORE|T6|Figueira|5|10|CADUCA|GERARFOLHAS
! Gestão de Habitats
  1 - Visualizar todos os habitats
  2 - Registar habitat
  3 - Alterar área de habitat
--- 18,24 ----
  5 - Plantar árvore
  6 - Mostrar árvores de habitat
  0 - Sair
! Escolha uma opção: Gestão de Habitats
  1 - Visualizar todos os habitats
  2 - Registar habitat
  3 - Alterar área de habitat
***************
*** 51,58 ****
  4 - Retirar responsabilidade
  5 - Calcular satisfação de funcionário
  0 - Sair
! Escolha uma opção: TRT|W2SO4|John Figueiredo|H1,H2
! Gestão de Funcionários
  1 - Visualizar todos os funcionários
  2 - Registar funcionário
  3 - Atribuir responsabilidade
--- 45,51 ----
  4 - Retirar responsabilidade
  5 - Calcular satisfação de funcionário
  0 - Sair
! Escolha uma opção: Gestão de Funcionários
  1 - Visualizar todos os funcionários
  2 - Registar funcionário
  3 - Atribuir responsabilidade
