*** tests/A-01-14-M-ok.out	2024-10-23 11:20:49.111157000 +0100
--- tests/A-01-14-M-ok.outhyp	2024-10-23 21:34:33.482392926 +0100
***************
*** 10,16 ****
  9 - Gestão de vacinas
  10 - Consultas
  0 - Sair
! Escolha uma opção: Ficheiro a abrir: Menu Principal
  1 - Criar ficheiro
  2 - Abrir ficheiro
  3 - Guardar ficheiro
--- 10,17 ----
  9 - Gestão de vacinas
  10 - Consultas
  0 - Sair
! Escolha uma opção: Ficheiro a abrir: Abrir ficheiro: Operação inválida: Problema ao abrir ficheiro: Erro a processar ficheiro ap01.dat
! Menu Principal
  1 - Criar ficheiro
  2 - Abrir ficheiro
  3 - Guardar ficheiro
***************
*** 28,40 ****
  3 - Transferir para habitat
  4 - Calcular satisfação de animal
  0 - Sair
! Escolha uma opção: ANIMAL|A1|Farrusco 1|E1|VOID|H1
! ANIMAL|A2|Farrusco 2|E2|VOID|H1
! ANIMAL|A3|Farrusco 3|E3|VOID|H2
! ANIMAL|B1|Farrusco 4|E3|VOID|H2
! ANIMAL|X1|Farrusco 5|E2|VOID|H3
! ANIMAL|X2|Farrusco 6|E1|VOID|H3
! Gestão de Animais
  1 - Visualizar todos os animais
  2 - Registar animal
  3 - Transferir para habitat
--- 29,35 ----
  3 - Transferir para habitat
  4 - Calcular satisfação de animal
  0 - Sair
! Escolha uma opção: Gestão de Animais
  1 - Visualizar todos os animais
  2 - Registar animal
  3 - Transferir para habitat
***************
*** 60,69 ****
  5 - Plantar árvore
  6 - Mostrar árvores de habitat
  0 - Sair
! Escolha uma opção: HABITAT|H1|ldeia dos Macacos|20|0
! HABITAT|H2|ldeia dos Macacos|20|0
! HABITAT|H3|ldeia dos Macacos|20|0
! Gestão de Habitats
  1 - Visualizar todos os habitats
  2 - Registar habitat
  3 - Alterar área de habitat
--- 55,61 ----
  5 - Plantar árvore
  6 - Mostrar árvores de habitat
  0 - Sair
! Escolha uma opção: Gestão de Habitats
  1 - Visualizar todos os habitats
  2 - Registar habitat
  3 - Alterar área de habitat
