*** tests/A-02-06-M-ok.out	2024-10-18 19:27:35.676097000 +0100
--- tests/A-02-06-M-ok.outhyp	2024-10-20 17:04:18.879573310 +0100
***************
*** 18,31 ****
  5 - Plantar árvore
  6 - Mostrar árvores de habitat
  0 - Sair
! Escolha uma opção: HABITAT|h1|nome2|12|2
! ÁRVORE|T1|Pinheiro 4|60|20|PERENE|GERARFOLHAS
! ÁRVORE|T3|Figueira|5|10|CADUCA|GERARFOLHAS
! HABITAT|h2|outro nome|122|0
! HABITAT|h3|nome1|1222|2
! ÁRVORE|T2|Oliveira|120|10|PERENE|GERARFOLHAS
! ÁRVORE|T4|Plátano|30|20|CADUCA|GERARFOLHAS
! Gestão de Habitats
  1 - Visualizar todos os habitats
  2 - Registar habitat
  3 - Alterar área de habitat
--- 18,24 ----
  5 - Plantar árvore
  6 - Mostrar árvores de habitat
  0 - Sair
! Escolha uma opção: Gestão de Habitats
  1 - Visualizar todos os habitats
  2 - Registar habitat
  3 - Alterar área de habitat
