*** tests/A-06-03-M-ok.out	2024-10-18 19:27:35.668087000 +0100
--- tests/A-06-03-M-ok.outhyp	2024-10-22 17:39:18.739232635 +0100
***************
*** 22,29 ****
  9 - Gestão de vacinas
  10 - Consultas
  0 - Sair
! Escolha uma opção: 3
! Menu Principal
  1 - Criar ficheiro
  2 - Abrir ficheiro
  3 - Guardar ficheiro
--- 22,28 ----
  9 - Gestão de vacinas
  10 - Consultas
  0 - Sair
! Escolha uma opção: Menu Principal
  1 - Criar ficheiro
  2 - Abrir ficheiro
  3 - Guardar ficheiro
