*** tests/A-09-03-M-ok.out	2024-10-18 19:27:35.656702000 +0100
--- tests/A-09-03-M-ok.outhyp	2024-10-22 17:39:19.543238238 +0100
***************
*** 22,35 ****
  3 - Transferir para habitat
  4 - Calcular satisfação de animal
  0 - Sair
! Escolha uma opção: Identificador único do animal: Nome do animal: Identificador único da espécie: Identificador único do habitat: Registar animal: Operação inválida: O animal 'a2' já existe.
! Gestão de Animais
  1 - Visualizar todos os animais
  2 - Registar animal
  3 - Transferir para habitat
  4 - Calcular satisfação de animal
  0 - Sair
! Escolha uma opção: ANIMAL|a2|animal 2|C10|VOID|h1
  Gestão de Animais
  1 - Visualizar todos os animais
  2 - Registar animal
--- 22,34 ----
  3 - Transferir para habitat
  4 - Calcular satisfação de animal
  0 - Sair
! Escolha uma opção: Identificador único do animal: Nome do animal: Identificador único da espécie: Identificador único do habitat: Gestão de Animais
  1 - Visualizar todos os animais
  2 - Registar animal
  3 - Transferir para habitat
  4 - Calcular satisfação de animal
  0 - Sair
! Escolha uma opção: ANIMAL|a2|outro|C10|VOID|h1
  Gestão de Animais
  1 - Visualizar todos os animais
  2 - Registar animal
