*** tests/A-10-04-M-ok.out	2024-10-23 11:20:49.111157000 +0100
--- tests/A-10-04-M-ok.outhyp	2024-10-23 21:34:36.762511935 +0100
***************
*** 16,22 ****
  3 - Transferir para habitat
  4 - Calcular satisfação de animal
  0 - Sair
! Escolha uma opção: Identificador único do animal: Identificador único do habitat: Gestão de Animais
  1 - Visualizar todos os animais
  2 - Registar animal
  3 - Transferir para habitat
--- 16,22 ----
  3 - Transferir para habitat
  4 - Calcular satisfação de animal
  0 - Sair
! Escolha uma opção: Identificador único do habitat: Identificador único do animal: Gestão de Animais
  1 - Visualizar todos os animais
  2 - Registar animal
  3 - Transferir para habitat
***************
*** 24,30 ****
  0 - Sair
  Escolha uma opção: ANIMAL|A1|Pinguim 1|C1|VOID|AR2
  ANIMAL|A2|Pinguim 2|C1|VOID|AR2
! ANIMAL|A3|Macaco 1|C2|VOID|AR1
  ANIMAL|A4|Macaco 2|C2|VOID|AR2
  Gestão de Animais
  1 - Visualizar todos os animais
--- 24,30 ----
  0 - Sair
  Escolha uma opção: ANIMAL|A1|Pinguim 1|C1|VOID|AR2
  ANIMAL|A2|Pinguim 2|C1|VOID|AR2
! ANIMAL|A3|Macaco 1|C2|VOID|AR2
  ANIMAL|A4|Macaco 2|C2|VOID|AR2
  Gestão de Animais
  1 - Visualizar todos os animais
***************
*** 50,64 ****
  3 - Actos médicos de um veterinário
  4 - Vacinações com problemas
  0 - Sair
! Escolha uma opção: Identificador único do habitat: ANIMAL|A3|Macaco 1|C2|VOID|AR1
! Consultas
  1 - Animais de um habitat
  2 - Actos médicos num animal
  3 - Actos médicos de um veterinário
  4 - Vacinações com problemas
  0 - Sair
  Escolha uma opção: Identificador único do habitat: ANIMAL|A1|Pinguim 1|C1|VOID|AR2
  ANIMAL|A2|Pinguim 2|C1|VOID|AR2
  ANIMAL|A4|Macaco 2|C2|VOID|AR2
  Consultas
  1 - Animais de um habitat
--- 50,68 ----
  3 - Actos médicos de um veterinário
  4 - Vacinações com problemas
  0 - Sair
! Escolha uma opção: Identificador único do habitat: Consultas
  1 - Animais de um habitat
  2 - Actos médicos num animal
  3 - Actos médicos de um veterinário
  4 - Vacinações com problemas
  0 - Sair
  Escolha uma opção: Identificador único do habitat: ANIMAL|A1|Pinguim 1|C1|VOID|AR2
+ ANIMAL|A1|Pinguim 1|C1|VOID|AR2
+ ANIMAL|A2|Pinguim 2|C1|VOID|AR2
  ANIMAL|A2|Pinguim 2|C1|VOID|AR2
+ ANIMAL|A3|Macaco 1|C2|VOID|AR2
+ ANIMAL|A3|Macaco 1|C2|VOID|AR2
+ ANIMAL|A4|Macaco 2|C2|VOID|AR2
  ANIMAL|A4|Macaco 2|C2|VOID|AR2
  Consultas
  1 - Animais de um habitat
***************
*** 84,90 ****
  3 - Transferir para habitat
  4 - Calcular satisfação de animal
  0 - Sair
! Escolha uma opção: Identificador único do animal: Identificador único do habitat: Gestão de Animais
  1 - Visualizar todos os animais
  2 - Registar animal
  3 - Transferir para habitat
--- 88,94 ----
  3 - Transferir para habitat
  4 - Calcular satisfação de animal
  0 - Sair
! Escolha uma opção: Identificador único do habitat: Identificador único do animal: Gestão de Animais
  1 - Visualizar todos os animais
  2 - Registar animal
  3 - Transferir para habitat
