*** tests/A-11-05-M-ok.out	2024-10-18 19:27:35.645823000 +0100
--- tests/A-11-05-M-ok.outhyp	2024-10-22 17:39:20.072241925 +0100
***************
*** 18,24 ****
  5 - Plantar árvore
  6 - Mostrar árvores de habitat
  0 - Sair
! Escolha uma opção: Identificador único do habitat: Identificador único da espécie: Influência (positiva, negativa, neutra: POS, NEG, NEU): Gestão de Habitats
  1 - Visualizar todos os habitats
  2 - Registar habitat
  3 - Alterar área de habitat
--- 18,24 ----
  5 - Plantar árvore
  6 - Mostrar árvores de habitat
  0 - Sair
! Escolha uma opção: Gestão de Habitats
  1 - Visualizar todos os habitats
  2 - Registar habitat
  3 - Alterar área de habitat
***************
*** 26,31 ****
--- 26,34 ----
  5 - Plantar árvore
  6 - Mostrar árvores de habitat
  0 - Sair
+ Escolha uma opção: Número inválido!
+ Escolha uma opção: Número inválido!
+ Escolha uma opção: Número inválido!
  Escolha uma opção: Menu Principal
  1 - Criar ficheiro
  2 - Abrir ficheiro
***************
*** 44,56 ****
  3 - Transferir para habitat
  4 - Calcular satisfação de animal
  0 - Sair
! Escolha uma opção: Identificador único do animal: 52
! Gestão de Animais
  1 - Visualizar todos os animais
  2 - Registar animal
  3 - Transferir para habitat
  4 - Calcular satisfação de animal
  0 - Sair
  Escolha uma opção: Menu Principal
  1 - Criar ficheiro
  2 - Abrir ficheiro
--- 47,59 ----
  3 - Transferir para habitat
  4 - Calcular satisfação de animal
  0 - Sair
! Escolha uma opção: Gestão de Animais
  1 - Visualizar todos os animais
  2 - Registar animal
  3 - Transferir para habitat
  4 - Calcular satisfação de animal
  0 - Sair
+ Escolha uma opção: Número inválido!
  Escolha uma opção: Menu Principal
  1 - Criar ficheiro
  2 - Abrir ficheiro
***************
*** 71,77 ****
  5 - Plantar árvore
  6 - Mostrar árvores de habitat
  0 - Sair
! Escolha uma opção: Identificador único do habitat: Identificador único da espécie: Influência (positiva, negativa, neutra: POS, NEG, NEU): Gestão de Habitats
  1 - Visualizar todos os habitats
  2 - Registar habitat
  3 - Alterar área de habitat
--- 74,80 ----
  5 - Plantar árvore
  6 - Mostrar árvores de habitat
  0 - Sair
! Escolha uma opção: Gestão de Habitats
  1 - Visualizar todos os habitats
  2 - Registar habitat
  3 - Alterar área de habitat
***************
*** 79,84 ****
--- 82,90 ----
  5 - Plantar árvore
  6 - Mostrar árvores de habitat
  0 - Sair
+ Escolha uma opção: Número inválido!
+ Escolha uma opção: Número inválido!
+ Escolha uma opção: Número inválido!
  Escolha uma opção: Menu Principal
  1 - Criar ficheiro
  2 - Abrir ficheiro
***************
*** 97,109 ****
  3 - Transferir para habitat
  4 - Calcular satisfação de animal
  0 - Sair
! Escolha uma opção: Identificador único do animal: 32
! Gestão de Animais
  1 - Visualizar todos os animais
  2 - Registar animal
  3 - Transferir para habitat
  4 - Calcular satisfação de animal
  0 - Sair
  Escolha uma opção: Menu Principal
  1 - Criar ficheiro
  2 - Abrir ficheiro
--- 103,115 ----
  3 - Transferir para habitat
  4 - Calcular satisfação de animal
  0 - Sair
! Escolha uma opção: Gestão de Animais
  1 - Visualizar todos os animais
  2 - Registar animal
  3 - Transferir para habitat
  4 - Calcular satisfação de animal
  0 - Sair
+ Escolha uma opção: Número inválido!
  Escolha uma opção: Menu Principal
  1 - Criar ficheiro
  2 - Abrir ficheiro
