*** tests/A-17-08-M-ok.out	2024-10-18 19:27:35.639324000 +0100
--- tests/A-17-08-M-ok.outhyp	2024-10-22 17:39:20.833247230 +0100
***************
*** 19,25 ****
  0 - Sair
  Escolha uma opção: TRT|T1|Trt one|H1,H2,H3
  TRT|T2|Trt two|H1,H3
! VET|V1|Vet one|C1,C3,C4
  VET|V2|Vet two
  Gestão de Funcionários
  1 - Visualizar todos os funcionários
--- 19,25 ----
  0 - Sair
  Escolha uma opção: TRT|T1|Trt one|H1,H2,H3
  TRT|T2|Trt two|H1,H3
! VET|V1|Vet one|C3,C4,C1
  VET|V2|Vet two
  Gestão de Funcionários
  1 - Visualizar todos os funcionários
