*** tests/A-18-05-M-ok.out	2024-10-18 19:27:35.634745000 +0100
--- tests/A-18-05-M-ok.outhyp	2024-10-20 17:04:21.635573300 +0100
***************
*** 26,40 ****
  4 - Retirar responsabilidade
  5 - Calcular satisfação de funcionário
  0 - Sair
! Escolha uma opção: Identificador único do funcionário: Identificador único da responsabilidade: Gestão de Funcionários
  1 - Visualizar todos os funcionários
  2 - Registar funcionário
  3 - Atribuir responsabilidade
  4 - Retirar responsabilidade
  5 - Calcular satisfação de funcionário
  0 - Sair
  Escolha uma opção: TRT|T1|Trt one
! VET|V1|Vet one|C1
  Gestão de Funcionários
  1 - Visualizar todos os funcionários
  2 - Registar funcionário
--- 26,42 ----
  4 - Retirar responsabilidade
  5 - Calcular satisfação de funcionário
  0 - Sair
! Escolha uma opção: Gestão de Funcionários
  1 - Visualizar todos os funcionários
  2 - Registar funcionário
  3 - Atribuir responsabilidade
  4 - Retirar responsabilidade
  5 - Calcular satisfação de funcionário
  0 - Sair
+ Escolha uma opção: Número inválido!
+ Escolha uma opção: Número inválido!
  Escolha uma opção: TRT|T1|Trt one
! VET|V1|Vet one|C1,C2
  Gestão de Funcionários
  1 - Visualizar todos os funcionários
  2 - Registar funcionário
