*** tests/A-18-04-M-ok.out	2024-10-23 11:20:49.111157000 +0100
--- tests/A-18-04-M-ok.outhyp	2024-10-23 21:34:37.962555683 +0100
***************
*** 33,46 ****
  4 - Retirar responsabilidade
  5 - Calcular satisfação de funcionário
  0 - Sair
! Escolha uma opção: Identificador único do funcionário: Identificador único da responsabilidade: Gestão de Funcionários
  1 - Visualizar todos os funcionários
  2 - Registar funcionário
  3 - Atribuir responsabilidade
  4 - Retirar responsabilidade
  5 - Calcular satisfação de funcionário
  0 - Sair
! Escolha uma opção: TRT|T1|Trt one
  VET|V1|Vet one
  Gestão de Funcionários
  1 - Visualizar todos os funcionários
--- 33,48 ----
  4 - Retirar responsabilidade
  5 - Calcular satisfação de funcionário
  0 - Sair
! Escolha uma opção: Gestão de Funcionários
  1 - Visualizar todos os funcionários
  2 - Registar funcionário
  3 - Atribuir responsabilidade
  4 - Retirar responsabilidade
  5 - Calcular satisfação de funcionário
  0 - Sair
! Escolha uma opção: Número inválido!
! Escolha uma opção: Número inválido!
! Escolha uma opção: TRT|T1|Trt one|H1
  VET|V1|Vet one
  Gestão de Funcionários
  1 - Visualizar todos os funcionários
