*** tests/A-21-10-M-ok.out	2024-10-18 19:27:35.614390000 +0100
--- tests/A-21-10-M-ok.outhyp	2024-10-20 17:04:22.730573296 +0100
***************
*** 17,37 ****
  4 - Mostrar vacinações
  0 - Sair
  Escolha uma opção: Número inválido!
! Escolha uma opção: Identificador da vacina: Identficador do veterinário a ministrar a vacina: Identificador único do animal: Gestão de Vacinas
  1 - Visualizar todas as vacinas
  2 - Registar vacina
  3 - Vacinar animal
  4 - Mostrar vacinações
  0 - Sair
! Escolha uma opção: Identificador da vacina: Identficador do veterinário a ministrar a vacina: Identificador único do animal: Gestão de Vacinas
  1 - Visualizar todas as vacinas
  2 - Registar vacina
  3 - Vacinar animal
  4 - Mostrar vacinações
  0 - Sair
! Escolha uma opção: REGISTO-VACINA|Vac1|Vet3|E1
! REGISTO-VACINA|Vac3|Vet3|E2
! Gestão de Vacinas
  1 - Visualizar todas as vacinas
  2 - Registar vacina
  3 - Vacinar animal
--- 17,41 ----
  4 - Mostrar vacinações
  0 - Sair
  Escolha uma opção: Número inválido!
! Escolha uma opção: Gestão de Vacinas
  1 - Visualizar todas as vacinas
  2 - Registar vacina
  3 - Vacinar animal
  4 - Mostrar vacinações
  0 - Sair
! Escolha uma opção: Número inválido!
! Escolha uma opção: Número inválido!
! Escolha uma opção: Número inválido!
! Escolha uma opção: Gestão de Vacinas
  1 - Visualizar todas as vacinas
  2 - Registar vacina
  3 - Vacinar animal
  4 - Mostrar vacinações
  0 - Sair
! Escolha uma opção: Número inválido!
! Escolha uma opção: Número inválido!
! Escolha uma opção: Número inválido!
! Escolha uma opção: Gestão de Vacinas
  1 - Visualizar todas as vacinas
  2 - Registar vacina
  3 - Vacinar animal
***************
*** 56,63 ****
  4 - Calcular satisfação de animal
  0 - Sair
  Escolha uma opção: ANIMAL|AA1|Patudo|E1|VOID|H1
! ANIMAL|AA2|Patudo 2|E1|NORMAL|H1
! ANIMAL|AA3|Malhado|E2|NORMAL|H1
  Gestão de Animais
  1 - Visualizar todos os animais
  2 - Registar animal
--- 60,67 ----
  4 - Calcular satisfação de animal
  0 - Sair
  Escolha uma opção: ANIMAL|AA1|Patudo|E1|VOID|H1
! ANIMAL|AA2|Patudo 2|E1|VOID|H1
! ANIMAL|AA3|Malhado|E2|VOID|H1
  Gestão de Animais
  1 - Visualizar todos os animais
  2 - Registar animal
***************
*** 82,90 ****
  3 - Vacinar animal
  4 - Mostrar vacinações
  0 - Sair
! Escolha uma opção: VACINA|Vac1|nome 1|1|E1
  VACINA|Vac2|nome 2|0|E2
! VACINA|Vac3|nome 3|1|E1,E2
  Gestão de Vacinas
  1 - Visualizar todas as vacinas
  2 - Registar vacina
--- 86,94 ----
  3 - Vacinar animal
  4 - Mostrar vacinações
  0 - Sair
! Escolha uma opção: VACINA|Vac1|nome 1|0|E1
  VACINA|Vac2|nome 2|0|E2
! VACINA|Vac3|nome 3|0|E1,E2
  Gestão de Vacinas
  1 - Visualizar todas as vacinas
  2 - Registar vacina
