*** testsBoth/A-01-18-M-ok.out	2024-10-03 22:33:10.825420000 +0100
--- testsBoth/A-01-18-M-ok.outhyp	2024-10-08 22:06:24.662486484 +0100
***************
*** 20,33 ****
  0 - Sair
  Escolha uma opção: 1
  TRT|A24|Rohit Figueiredo
- VET|A25|Rohit Figueiredo 2
- TRT|A34|Rohit Figueiredo 3
- VET|A85|Rohit Figueiredo 4
- VET|A95|Rohit Figueiredo 5
- TRT|B24|Rohit Figueiredo 6
- TRT|D24|Rohit Figueiredo 7
- TRT|X24|Rohit Figueiredo 8
- VET|X25|Rohit Figueiredo 9
  Gestão de Funcionários
  1 - Visualizar todos os funcionários
  2 - Registar funcionário
--- 20,25 ----
