*** testsBoth/A-01-16-M-ok.out	2024-10-03 22:33:10.650519000 +0100
--- testsBoth/A-01-16-M-ok.outhyp	2024-10-07 16:42:10.383928874 +0100
***************
*** 19,29 ****
  5 - Calcular satisfação de funcionário
  0 - Sair
  Escolha uma opção: 1
- TRT|A24|Rohit Figueiredo
- TRT|A34|Rohit Figueiredo 2
- TRT|B24|Rohit Figueiredo 3
- TRT|D24|Rohit Figueiredo 4
- TRT|X24|Rohit Figueiredo 5
  Gestão de Funcionários
  1 - Visualizar todos os funcionários
  2 - Registar funcionário
--- 19,24 ----
