*** testsBoth/A-01-17-M-ok.out	2024-10-03 22:33:10.734237000 +0100
--- testsBoth/A-01-17-M-ok.outhyp	2024-10-08 22:06:24.602486981 +0100
***************
*** 19,29 ****
  5 - Calcular satisfação de funcionário
  0 - Sair
  Escolha uma opção: 1
- VET|A24|Rohit Figueiredo
- VET|A34|Rohit Figueiredo 2
- VET|B24|Rohit Figueiredo 3
- VET|D24|Rohit Figueiredo 4
- VET|X24|Rohit Figueiredo 5
  Gestão de Funcionários
  1 - Visualizar todos os funcionários
  2 - Registar funcionário
--- 19,24 ----
